sample-app/EmployeeDirectory/package.json

```diff
@@ -7,7 +7,7 @@
 		"test": "jest"
 	},
 	"dependencies": {
-		"cavy": "^0.1.0",
+		"cavy": "^0.4.1",
```

Comment:
It's possible to knock out https://github.com/pixielabs/cavy/issues/55 at the same time. Not sure if it's totally out of scope for this change.